### DISCUSSION TITLE
mod ain't working

### DISCUSSION THREAD

**Discussion by Noirling** [thatskymod/Sky-CotL-Scripts#105] [Q&A]
hello! I have been having problems to make the mod work for weeks since the lastest season update, even I updated the langit mod it seem to not work still. When I try to CR it gets stuck at %1 and the app seem to not read the friend list as well so I can't get the ligts automatically either. I checked the langit mod got updated 5 days ago I deleted and redownloaded it but it still doesn't work.. any help to fix please? :')

**Comment by Mayone217** [comment]
Is it working now? I still can't use it.

**Comment by thatskymod** [comment]
I can infer that this thread relates to an earlier version of Sky, 0.26.2.

Upon inspection, it was determined that the following versions are compatible and operational:
- libLangit043.so (Live + Beta)
- libRomanBeta.so 2.1.0 (Live only)

Outdated and incompatible versions have been removed from this collection for the sake of clarity.
